package.json

```diff
@@ -41,7 +41,8 @@
     "prettier": "^2.7.1"
   },
   "dependencies": {
-    "@project-serum/anchor": "^0.24.2",
+    "@metaplex-foundation/mpl-token-metadata": "^2.2.4",
+    "@project-serum/anchor": "^0.25.0",
```

Comment:
I'm hesitant to bump to this version until raindrops has some more automated testing in place.

Reply:
I don't want to upgrade anchor version because the v0.24.2 doesn't have associated token program in anchor spl.
If we keep using anchor v0.24.2, we need to install extra spl-token package to support associated token program.

Reply:
I will keep the v0.24.2 version then.

Reply:
Hmm, actually don't need to add `@solana/spl-token` as an extra package in `package.json` because metaplex package includes it. Go to keep using anchor v0.24.2 then.